bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/helpers/RepositoryHelper.java

```diff
@@ -138,4 +144,36 @@ public static <T> IRepository<T> createMemoryComposite(IProvisioningAgent agent,
 		return null;
 	}
 
+	/**
+	 * Returns an unmodifiable list of global shared bundle pools, e.g., as used by
+	 * the Eclipse Installer.
+	 *
+	 * @return an unmodifiable list of global shared bundle pools.
+	 *
+	 * @throws IOException if there are problems loading the pool information.
+	 */
+	@SuppressWarnings("nls")
+	public static List<Path> getSharedBundlePools() throws IOException {
+			Path bundlePools = Path.of(System.getProperty("user.home"), ".p2/pools.info");
```

Comment:
Indentation looks wrong here

Reply:
Feel free to improve things.  This often happens because only touched lines are formatted, and I removed the if guard.  Sorry about that.